src/useCases/getFilteredUsers.ts

```diff
@@ -7,10 +7,15 @@ import UserDetails from "types/UserDetails"
 const getFilteredUsers = async (
   connection: Database,
   filter: string,
+  visibleForces: string,
   page = 0
 ): PromiseResult<PaginatedResult<Partial<UserDetails>[]>> => {
-  let users
-  const getFilteredUsersQuery = `
+  let users = []
+  if (visibleForces !== "") {
+    const forces = visibleForces.split(",")
+    /* eslint-disable @typescript-eslint/naming-convention */
+    const forceWhere = forces.map((code) => `visible_forces like '%${code}%'`).join(" OR ")
```

Comment:
Same point as above - we should use something stricter than a wildcard (`%`) here.